composer.json

```diff
@@ -25,6 +25,9 @@
 		"data-values/interfaces": "~0.2.0|~0.1.5",
 		"data-values/common": "~0.3.0|~0.2.0"
 	},
+	"require-dev": {
+		"phpunit/phpunit": "~4.8"
```

Comment:
I prefer for all our libraries to use the same version of tools and libraries. Otherwise, it becomes confusing what feature you can use where. If we want to move to 5.x, let's do that for all our stuff.